app/models/external_media_resource.rb

```diff
@@ -0,0 +1,34 @@
+class ExternalMediaResource < MediaResource
+  validates :duration, numericality: {greater_than: 0}, if: :status_complete?
+  after_create_commit :analyze_media
+
+  def guid
+    nil
+  end
+
+  def url
+    original_url
+  end
+
+  def media_url
+    original_url
+  end
+
+  def copy_media(force = false)
+  end
+
+  def analyze_media(force = false)
```

Comment:
Can we keep using copy_media (even if the name is misleading).  We could pretty easily rename across the board to `.process_media` if we wanted.